email/1.3/modules/ROOT/pages/email-list.adoc

```diff
@@ -1,203 +1,209 @@
 = List Emails with Email Connector Examples - Mule 4
 :page-aliases: connectors::email/email-list.adoc
 
-Email Connector can list emails from any part of the flow. This capability is a change from the Mule 3 transport, which could retrieve emails only as a result of inbound endpoint polling.
+Anypoint for Email (Email Connector) provides the *List - IMAP* and *List - POP3* operations to list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector.
+
+* <<send-attachments,List Emails from an IMAPS Server>> +
+Configure the *List - IMAP* operation to list all the emails in the configured IMAP mailbox folder from an IMAPS server with TLS connection. Additionally, set a *For Each* scope and *Set Variable* components to iterate over the emails and save the email body and attachment content in variables.
+
+* <<send-attachments,List Emails from a POP3 Server>> +
+Configure the *List - POP3* operation to list all the emails in the configured POP3 mailbox folder that matches with the specified *Pop3 matcher* criteria.
+
+These example uses variables for some field values. You can either: +
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to that file from the connector configuration. +
+If you don’t know how to use a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+== List Emails from an IMAPS Server
+
+The following example illustrates how to list all the emails in the configured IMAP mailbox folder from an IMAPS server with TLS connection. The flow initiates with a *Scheduler* component, and then the *List - IMAP* operation list the emails. Subsequently, a *For Each* scope and *Set Variable* components iterate over the emails, and save the body and attachments of the email content payload in variables:
```

Comment:
Same comment as before about the TLS connection

Suggested rewording: A *Scheduler* component initiates the flow, and then the *List - IMAP* operation lists the emails.